client/src/charts/legends/StandardLegend.scss

```diff
@@ -7,3 +7,9 @@
   border-radius: $standardBorderRadius;
   margin-bottom: 4px;
 }
+
+.title-p--standard-legend {
```

Comment:
Proper BEM would be something like `standard-legend-container__title`

Think of BEM as block__element--modifier. Block is roughly analogous to the React element itself. Element is some specific sub-element of that (preferably identifiable as a concrete concept like "title"). Modifier is some variable state effecting either a block or element's style, such as `--active` etc.

The BEM names do not need to, and generally _shouldn't_, represent the actual corresponding HTML structure. HTML structure can be explicitly represented in the selectors if necessary. Otherwise, the BEM class names are a bit of an abstraction over top of the actual HTML implementation. In theory, there could be plenty of cases where HTML structure is edited but the corresponding BEM class names remain the same. E.g. the title could be moved around within the legend container or be changed from a `p` to a `div` or `a`, but it would still be, conceptually, a "title belonging to the legend container" and likely should still be styled the same.

Definitely not advised to ever put an html tag, like `p`, in the class name. If it's actually important to the selector, than you would write it as something like `p.<class name>` rather than `.<class name mentioning p>`.

Reply:
Ok thanks for the advice!